package/features/steps/lower_requirements.txt

```diff
@@ -13,4 +13,3 @@ strawberry-graphql==0.192.0
 networkx==2.5
 orjson==3.9
 secure==0.3.0
-pydantic==1.10
```

Comment:
This seems different from the discussion in https://github.com/kedro-org/kedro-viz/issues/1603. Just to confirm, we are dropping Pydantic v1 support altogether now.

Reply:
yes we are dropping support for pydantic v1 as there are issues with the v1 api mentioned [here](https://github.com/tiangolo/fastapi/issues/10360) and an open PR - https://github.com/tiangolo/fastapi/pull/10223/files